cranelift/codegen/src/machinst/lower.rs

```diff
@@ -93,6 +94,8 @@ pub trait LowerCtx {
     /// every side-effecting op; the backend should not try to merge across
     /// side-effect colors unless the op being merged is known to be pure.
     fn inst_color(&self, ir_inst: Inst) -> InstColor;
+    /// Determine whether an instruction is a safepoint.
+    fn is_safepoint(&self, ir_inst: Inst) -> bool;
```

Comment:
How is this used?  In particular, will we have to call `is_safepoint` on every insn that is processed?  That would seem to be an undesirable compiler performance hit, if so?  (Tell me it ain't so!)

Reply:
Sorry, this was left over from an earlier iteration that did indeed call `is_safepoint` on every instruction; but we no longer do that, and the function was dead, so I removed it.